### PR TITLE
Added identityVerified param to organizations query + added identityVerified toggle mutation

### PR DESCRIPTION
**Pull request by jeremyzitomer-usds**
## Related Issue or Background Info

- backend for https://app.zenhub.com/workspaces/simplereport-growth--engagement-605b43a6ceb92c000f86279a/issues/cdcgov/prime-simplereport/1317

## Changes Proposed

- added nullable `identityVerified` filter to existing `organizations` query
- created `setOrganizationIdentityVerified` mutation, plus tests

### CODE REVIEW

**Comment by jeremyzitomer-usds** [comment]
please forgive the Sonar failure for low test coverage, can't unit test the `LiveOktaRepository` code, otherwise all set to review

**Comment by brmaden** [comment]
Is the account activation email going to be done separately?  It looks like it's probably blocked waiting for content.

**Comment by jeremyzitomer-usds** [comment]
the email to activate a user's account is sent under the hood of user.activate() or user.reactivate(), i added a comment to clarify!

**Review by brmaden — APPROVED** [review]

File: backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationResolver.java [hunk 1]
```diff
@@ -32,13 +32,13 @@ public OrganizationResolver(OrganizationService os) {
         });
   }
 
-  public List<ApiOrganization> getOrganizations() {
+  public List<ApiOrganization> getOrganizations(Boolean identityVerified) {
```
Comment (benwarfield-usds) [inline backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationResolver.java:35]: I feel like some documentation of why this is an optional argument might be good here? Confused me initially.

File: backend/src/main/java/gov/cdc/usds/simplereport/db/repository/OrganizationRepository.java [hunk 2]
```diff
@@ -14,4 +14,7 @@
 
   @Query(EternalAuditedEntityRepository.BASE_QUERY + " and e.externalId in (:externalIds)")
   public List<Organization> findAllByExternalId(Collection<String> externalIds);
+
+  @Query(EternalAuditedEntityRepository.BASE_QUERY + " and e.identityVerified = :identityVerified")
+  public List<Organization> findAllByIdentityVerified(Boolean identityVerified);
```
Comment (benwarfield-usds) [inline backend/src/main/java/gov/cdc/usds/simplereport/db/repository/OrganizationRepository.java:19]:
```suggestion
  public List<Organization> findAllByIdentityVerified(boolean identityVerified);
```

File: backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/OrganizationFacilityTest.java [hunk 3]
```diff
@@ -32,6 +36,41 @@ void createOrganization_siteAdminUser_ok() {
...
+            .put("verified", false);
+    runQuery("set-organization-identity-verified", variables, ACCESS_ERROR);
+  }
+
+  @Test
+  void setOrganizationIdentityVerified_siteAdminUser_ok() {
+    TestUserIdentities.withUser(
+        TestUserIdentities.SITE_ADMIN_USER,
+        () -> {
+          Organization org = _dataFactory.createValidOrg();
```
Comment (benwarfield-usds) [inline backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/OrganizationFacilityTest.java:54]: Only this line actually needs to be in the `withUser` block, no?

**Review by benwarfield-usds — COMMENTED** [review]
Looks like good stuff, had a couple minor notes to throw around while I'm catching up on PRs.

File: backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/OrganizationFacilityTest.java (hunk 3 above)
Comment (jeremyzitomer-usds) [reply backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/OrganizationFacilityTest.java:54]: that's what i thought initially, but then i have to reference `org` later in the code block, so i think it's all or nothing

Comment (jeremyzitomer-usds) [reply backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/OrganizationFacilityTest.java:54]: unless there's some secret Java way around this

**Pull request merged**